docs/antora-ccutil/README.md

```diff
@@ -0,0 +1,62 @@
+# Bundling documentation with Antora for ccutil/Pantheon
+
+This project uses Antora to prepare documentation to be used downstream with ccutil/Pantheon.
+
+## Components of the solution
+
+[Antora](https://antora.org/): Creating static websites with documentation from documentation stored in one or more Git repositories.
+In addition to that, providing APIs to publish from and to other sources and targets.
+
+[Antora Collector](https://gitlab.com/antora/antora-collector-extension): Acquire content for Antora from other sources than Git. 
+In this project, run a shell script to collect the documentation from the Keycloak Guides.
+
+[Antora Assembler](https://gitlab.com/antora/antora-assembler): Bundle contents as a single AsciiDoc file that can be used for processing with standard AsciiDoc tools.
+In this project, it generates the output for ccutil.
+There is a customized version in the `lib` folder which creates content in a way that is suitable for ccutil. 
+
+## Creating AsciiDoc content for downstream
+
+Antora needs to run inside the main working tree.
+Running it ina secondary working tree will fail as it won't find the `.git` folder.
+
+The process is wrapped with Maven, so to create the docs run from the root of the project:
+
+```bash
+mvn -am -pl docs/antora-ccutil clean install -DskipTests
+```
+
+As a result, the content for downstream is created as a ZIP file as `target/keycloak-antora-ccutil-${PROJECT_VERSION}-ccutil.zip`.
+The contents of the file contain all information and the correct structure to be processed by ccutil and Pantheon.
+
+## Additional commands
+
+Set the environment variable for the project version (normally done via Maven): 
+
+```bash
+export PROJECT_VERSION=999-SNAPSHOT
+```
+
+Install all npm dependencies to build the site locally:
+
+```bash
+npm install
+```
+
+Use the author playbook to generate a local HTML site for a preview:
+
+```bash
+npm run ccutil
+```
+
+Run the following script to process the contents of the ZIP with ccutil in a container to create single- and multipage output:
+
+```bash
+./ccutil.sh
+```
+
+Start a minimal webserver to preview the site locally.
+Once it has been started, go to http://localhost:5000/keycloak-benchmark/ to preview the contents.
+
+```bash
+npm run serve
+```
```

Comment:
Should we perhaps just remove this? Not sure why I'd want to know this ;)

Reply:
The first command help in development to create a partial build. To minimize the "surface", I'll reduce it to just "ccutil" and "npm run serve" as those come in handy and are not part (and shouldn't be part) of the regular build.

Reply:
minimized